Microsoft.DotNet.Interactive.FSharp/FSharpKernel.fs

```diff
@@ -228,8 +166,41 @@ type FSharpKernel() as this =
 
     override this.TryGetVariable(name: string, [<Out>] value: Object byref) =
         match this.GetCurrentVariable(name) with
-        | Some(cv) -> 
+        | Some(cv) ->
             value <- cv.Value
             true
-        | None -> 
+        | None ->
             false
+
+    interface ISupportNuget with
```

Comment:
Can we get the setters off of the interface and protected better against properties / fields ever getting set to null?

Reply:
I can go back to how it was before with an initialization method, certainly the properties are not necessary.  But I didn't like that very much, either, the tough part is the implementation of those guys, need to be in PackageRestoreContext, because that's where the data is.

Reply:
done